transformation/src/test/test-files/outputs/library/part-1.json

```diff
@@ -1,4 +1,4 @@
-{"id":"ENCLB652PHO","cross_references":[],"time_created":"2019-08-14T03:47:10.607313Z","award":"/awards/UM1HG009390/","lab":"/labs/bradley-bernstein/","size_range":"200-600","strand_specificity":false,"treatments":[],"submitted_by":"/users/f1843c60-e027-4b18-8582-64d3f3eae45b/","spike_ins":[],"biosample_id":"ENCBS280DYM"}
-{"id":"ENCLB957XQS","cross_references":[],"time_created":"2018-03-29T23:07:06.465239Z","award":"/awards/U54HG007005/","lab":"/labs/gene-yeo/","queried_rnp_size_range":"65-140","size_range":"175-300","strand_specificity":true,"treatments":[],"submitted_by":"/users/599e4901-78bf-422f-b65d-f3443c6a6450/","spike_ins":[],"biosample_id":"ENCBS768GCL"}
-{"id":"ENCLB943WFS","cross_references":["GEO:GSM2343999"],"time_created":"2014-12-11T15:52:17.464599Z","award":"/awards/U54HG007005/","lab":"/labs/brenton-graveley/","rna_integrity_number":9.4,"size_range":">200","strand_specificity":true,"treatments":[],"submitted_by":"/users/599e4901-78bf-422f-b65d-f3443c6a6450/","spike_ins":["/references/ENCSR449DXG/"],"biosample_id":"ENCBS575HTN"}
-{"id":"ENCLB555AOW","cross_references":[],"time_created":"2013-11-14T19:28:32.681368Z","award":"/awards/U54HG004557/","lab":"/labs/thomas-gingeras/","size_range":"<200","strand_specificity":true,"treatments":["/treatments/35c68491-07b7-4cc4-8eb1-6e13787a6d5a/"],"submitted_by":"/users/0abbd494-b852-433c-b360-93996f679dae/","spike_ins":[],"biosample_id":"ENCBS733AAA"}
+{"award":"/awards/U54HG004557/","biosample_id":"ENCBS733AAA","cross_references":[],"id":"ENCLB555AOW","lab":"/labs/thomas-gingeras/","size_range":"<200","spike_ins":[],"strand_specificity":true,"submitted_by":"/users/0abbd494-b852-433c-b360-93996f679dae/","time_created":"2013-11-14T19:28:32.681368Z","treatments":["/treatments/35c68491-07b7-4cc4-8eb1-6e13787a6d5a/"]}
+{"award":"/awards/U54HG007005/","biosample_id":"ENCBS575HTN","cross_references":["GEO:GSM2343999"],"id":"ENCLB943WFS","lab":"/labs/brenton-graveley/","rna_integrity_number":9.4,"size_range":">200","spike_ins":["/references/ENCSR449DXG/"],"strand_specificity":true,"submitted_by":"/users/599e4901-78bf-422f-b65d-f3443c6a6450/","time_created":"2014-12-11T15:52:17.464599Z","treatments":[]}
+{"award":"/awards/U54HG007005/","biosample_id":"ENCBS768GCL","cross_references":[],"id":"ENCLB957XQS","lab":"/labs/gene-yeo/","queried_rnp_size_range":"65-140","size_range":"175-300","spike_ins":[],"strand_specificity":true,"submitted_by":"/users/599e4901-78bf-422f-b65d-f3443c6a6450/","time_created":"2018-03-29T23:07:06.465239Z","treatments":[]}
+{"award":"/awards/UM1HG009390/","biosample_id":"ENCBS280DYM","cross_references":[],"id":"ENCLB652PHO","lab":"/labs/bradley-bernstein/","size_range":"200-600","spike_ins":[],"strand_specificity":false,"submitted_by":"/users/f1843c60-e027-4b18-8582-64d3f3eae45b/","time_created":"2019-08-14T03:47:10.607313Z","treatments":[]}
```

Comment:
In these outputs, are values just in a different order? Or did something else change?

Reply:
Yeah, the script to generate new outputs sorts by key.